core/unit_test/CMakeLists.txt

```diff
@@ -377,14 +377,23 @@ if(Kokkos_ENABLE_PTHREAD)
   )
 endif()
 
-if(Kokkos_ENABLE_OPENMP)
+if (Kokkos_ENABLE_OPENMP)
+  if (Kokkos_ENABLE_DEPRECATED_CODE_3)
```

Comment:
We should be really using `KOKKOS_` variables but I did not change for consistency